portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+var slideIndex = 0;
+showSlides();
+
+function showSlides() {
+  var i;
+  var slides = document.getElementsByClassName("individualSlide");
+  var dots = document.getElementsByClassName("dot");
+  for (i = 0; i < slides.length; i++) {
+    slides[i].style.display = "none";  
+  }
```

Comment:
nit: declare the loop variable `i` in each loop statement, to make it more clear that it should *only* be used in the loop

```js
for (var i = 0; i < slides.length; i++) {
  // ...
}
```